v2/src/css/custom.css

```diff
@@ -268,3 +270,96 @@ html[data-theme='dark'] .admonition.admonition-note {
 html[data-theme='dark'] .admonition .admonition-heading .admonition-icon {
   display: none;
 }
+
+/* left menu style updates */
+html[data-theme='dark'] aside > div > nav {
+  padding: 16px 0px 30px;
+}
+
+html[data-theme='dark'] aside > div > nav > .menu__list > .menu__list-item > .menu__link {
+  text-transform: uppercase;
+  font-size: 11px;
+  padding: 0px 16px;
+  margin: 12px 0 0px;
+  border-radius: 0;
+  color: rgb(188, 188, 188);
+  line-height: 16px;
+}
+
+html[data-theme='dark'] aside > div > nav .menu__list .menu__list-item {
+  margin-top: 0px;
+}
+
+html[data-theme='dark'] aside > div > nav .menu__list .menu__list-item .menu__link--sublist:after {
+  background: url("/img/docs-left-menu-chevron.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 8px 14px;
+  transform: rotate(90deg);
+  filter: none;
+  width: 8px;
+  height: 14px;
+}
+
+html[data-theme='dark'] aside > div > nav .menu__list .menu__list-item--collapsed .menu__link--sublist:after {
+  transform: rotate(0deg);
```

Comment:
What about Safari?

Reply:
Supported on Safari v9 and above - https://caniuse.com/?search=transform